NetflixRouletteWebpack/src/components/modals/edit-movie-modal/EditMovieModal.js

```diff
@@ -8,61 +11,61 @@ import crossIcon from "../close_button.png";
 import GenreDropdown from "../../genre-dropdown/GenreDropdown";
 
 const BASE_CLASS = "edit-movie-modal";
-
-const placeHolders = {
-  titlePlaceholder: "Moana",
-  movieUrlPlaceholder: "https://",
-  releaseDatePlaceholder: "Select Date",
-  ratingPlaceholder: "7.8",
-  runtimePlaceholder: "minutes",
-  overviewPlaceholder: "Movie description",
-};
 function EditMovieModal({
   handleEditModalOpen,
   handleMovieEdit,
   modalTitle,
   showCongratsModal,
-  editingValues,
+  initialValues,
 }) {
   const [selectedGenres, setSelectedGenres] = useState("");
-  const {
-    title: titleValue,
-    image: movieUrlValue,
-    releaseDate: releaseDateValue,
-    rating: ratingValue,
-    runtime: runtimeValue,
-    overview: overviewValue,
-    id,
-  } = editingValues;
-
-  const onFormSubmit = (e) => {
-    e.preventDefault();
-    const data = new FormData(e.target);
-
-    handleMovieEdit({
-      title: data.get("title"),
-      genre: selectedGenres,
-      releaseDate: data.get("release-date"),
-      rating: data.get("rating"),
-      runtime: data.get("runtime"),
-      image: data.get("movie-url"),
-      overview: data.get("overview"),
-      id,
-    });
-
-    if (modalTitle === "Add movie") {
-      showCongratsModal(true);
+  useEffect(() => {
+    if (modalTitle === "Edit movie") {
+      setSelectedGenres(initialValues.genre);
     }
-  };
+  }, [initialValues.genre, modalTitle]);
 
-  const {
-    titlePlaceholder,
-    movieUrlPlaceholder,
-    releaseDatePlaceholder,
-    ratingPlaceholder,
-    runtimePlaceholder,
-    overviewPlaceholder,
-  } = placeHolders;
+  const formik = useFormik({
+    initialValues,
+    validationSchema: Yup.object({
```

Comment:
@Marysosh Better move validation schema to the separate file

Reply:
Corrected✅